src/devices/src/virtio/net/unixgram.rs

```diff
@@ -126,7 +133,7 @@ impl NetBackend for Gvproxy {
     }
 
```

Comment:
another leftover passt in line 111. We can also clean this up later to keep this change minimal, these leftovers were already there.